easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -73,51 +72,23 @@
 # set of configure/build/install options that can be provided as lists for an iterated build
 ITERATE_OPTIONS = ['preconfigopts', 'configopts', 'prebuildopts', 'buildopts', 'preinstallopts', 'installopts']
 
-# map of deprecated easyconfig parameters, and their replacements
-DEPRECATED_OPTIONS = {
```

Comment:
so we will never have deprecated options again? i'd keep this code, just leave it empty.